app/controllers/idv/capture_doc_status_controller.rb

```diff
@@ -50,7 +50,11 @@ def document_capture_session
     end
 
     def throttled?
-      Throttler::IsThrottled.call(document_capture_session.user_id, :idv_acuant)
+      return unless document_capture_session.user
```

Comment:
Should it return a boolean? Is it necessary at all, given that I can't see that the previous logic would have guarded for this?

Reply:
I think we can get rid of the guard... 45c436fcb8e15b7b69a4ba4ee2e6ec2f47389313